### PR TITLE
Renaming TagsInput to ChipsInput

### PR DESCRIPTION
**Pull request by ArnonZ**
## Description
Renaming TagsInput to ChipsInput

## Changelog
Renaming TagsInput to ChipsInput

### CODE REVIEW

File: eslint-rules/tests/component_deprecation.json [hunk 1]
```diff
@@ -32,5 +32,11 @@
     "component": "List.Part",
     "source": "module-with-deprecations",
     "message": "Please use the 'List.Item' component instead."
+  },
+  {
+    "component": "TagsInput",
+    "source": "module-with-deprecations",
+    "message": "TagsInput was renamed, Please use 'ChipsInput' instead.",
+    "fix": {"componentName": "ChipsInput"}
   }
```
Comment (ethanshar) [inline eslint-rules/tests/component_deprecation.json]: These rules here are not going anywhere.. 
unlike our private lib when we use the rules for One App users. 
This file is just an example so there's no need to add a migration here.

File: src/index.js [hunk 2]
```diff
@@ -150,9 +150,12 @@ export default {
   get TabBar() {
     return require('./components/tabBar').default;
   },
-  get TagsInput() {
+  get TagsInput() { //Renamed to ChipsInput, to be deleted after migration
```
Comment (ethanshar) [inline src/index.js:153]:
```suggestion
  get TagsInput() { // TODO: Renamed to ChipsInput, to be deleted after migration
```

File: src/components/maskedInput/index.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ import Text from '../text';
 /**
  * @description: Mask Input to create custom looking inputs with custom formats
  * @extends: TextInput
- * @extendslink: docs/TagsInput
+ * @extendslink: docs/ChipsInput
```
Comment (ethanshar) [inline src/components/maskedInput/index.js:13]: This is actually wrong. 
maskedInput doesn't extend TagsInput
I assume it's an old copy-paste 
you can remove the `extendslink` entry

File: src/components/index.js [hunk 4]
```diff
@@ -134,10 +134,12 @@ export default {
   get TabBar() {
     return require('./tabBar').default;
   },
-  get TagsInput() {
+  get TagsInput() { //Renamed to ChipsInput, to be deleted after migration
```
Comment (ethanshar) [inline src/components/index.js:137]:
```suggestion
  get TagsInput() { // TODO: Renamed to ChipsInput, to be deleted after migration
```

**Review by ethanshar — CHANGES_REQUESTED** [review]

File: eslint-rules/tests/component_deprecation.json (hunk 1 above)
Comment (ArnonZ) [reply eslint-rules/tests/component_deprecation.json]: Yeah, I removed it :)

File: src/components/maskedInput/index.js (hunk 3 above)
Comment (ArnonZ) [reply src/components/maskedInput/index.js]: It does extend `TextInput`, so I updated it correctly.

**Review by ethanshar — APPROVED** [review]

**Pull request merged**